src/consts.js

```diff
@@ -9,6 +9,8 @@ module.exports = {
         appCmdQuit: "app.cmd.quit",
         appEventReady: "app.event.ready",
         appEventSecondInstance: "app.event.second.instance",
+        appExecuteJavaScript: "app.execute.javascript",
+        appExecuteJavaScriptCallback: "app.execute.javascript.callback",
```

Comment:
Could you rename that to `appEventExecutedJavascript: "app.event.executed.javascript"`?